test/views/reviews-view.test.js

```diff
@@ -113,6 +113,59 @@ describe('ReviewsView', function() {
     assert.lengthOf(wrapper.find('details.github-Review'), 2);
   });
 
+  it('displays an author association badge for review summaries', function() {
+    const {summaries, commentThreads} = aggregatedReviewsBuilder()
+      .addReviewSummary(r => r.id(0).authorAssociation('MEMBER'))
```

Comment:
Thanks for being thorough and testing all the associations!

I really want you to not have to type all these things out manually.  I wish it was possible to do something like...
```
const {summaries, commentThreads} = aggregatedReviewsBuilder();
let idx = 0;
Object.entries(authorAssociationText).forEach(([key, value]) => {
  addReviewSummary(r => r.id(idx).authorAssociation(value));
  idx ++;
});
summaries.build();
comments.build();
```

I don't think we can actually use the builders that way though.  

I mean, you could create a new review and summary for each association and then test those.  Since you already put in the effort to manually type all these out, not sure it's worth the time to optimize.